src/App.js

```diff
@@ -8,9 +8,11 @@ import React from 'react';
 import logo from './logo.png';
 import './App.css';
 import Dashboard from './components/Dashboard';
+import TopBar from './components/TopBar';
```

Comment:
I am not sure if the color of this Logo is good enough. Do you mind adding the original logo from huskyCI? 

![image](https://user-images.githubusercontent.com/8943477/66441005-76316800-ea0b-11e9-9cee-ced075fcf839.png)

By the way, if we use a dark background in our top bar, is it possible to have it in a "circle", just like we did it here? 

![image](https://user-images.githubusercontent.com/8943477/66441051-9cef9e80-ea0b-11e9-819d-90f85a2e2e17.png)

Reply:
Done.